### PR TITLE
Feature: allow rn reanimated v2 peer dependency

### PR DESCRIPTION
**Pull request by danielwinkler**
would be nice if the lib could exist along RN Reanimated 2 projects

This implements the migration guide
https://docs.swmansion.com/react-native-reanimated/docs/next/migration/
and also for redash
https://github.com/wcandillon/react-native-redash#%EF%B8%8F-v1-users-%EF%B8%8F

### CODE REVIEW

**Comment by gorhom** [comment]
hi @danielwinkler, thanks for submitting this pr , I will check it out later today :)

**Comment by danielwinkler** [comment]
For me the recommendation 
```json
"include": ["node_modules/react-native-redash/lib/typescript/v1/index.d.ts"]
```
did not work, so i put it to paths
Maybe you have a better idea how to fix it.

File: src/utilities/reanimated.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const {
+  Easing: EasingV1,
+  EasingNode: EasingV2,
+} = require('react-native-reanimated');
```
[the remark this answers is not in the document]
Comment (gorhom) [reply src/utilities/reanimated.ts:4]: both are doing the same thing, i am just trying to keep custom imports unified across repositories

**Review by gorhom — APPROVED** [review]

File: src/utilities/reanimated.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (danielwinkler) [reply src/utilities/reanimated.ts:4]: Thanks for the info!

**Pull request merged**